hypershift-operator/main.go

```diff
@@ -124,6 +126,8 @@ type StartOptions struct {
 	EnableUWMTelemetryRemoteWrite          bool
 	EnableValidatingWebhook                bool
 	EnableDedicatedRequestServingIsolation bool
+	EnableScaleFromZero                    bool
+	AWSCreds                               string // Path to mounted credentials file (set by deployment)
```

Comment:
These creds are specifically for the new controller, so I would name something specific like:
`scale-from-zero-aws-creds` or something similar.
(same for the flag name)